rpc/gnmi/gnmi-specification.md

```diff
@@ -1283,8 +1291,8 @@ root +
 In this case, nodes `k1` and `k2` are standalone nodes within the schema, but
 also correspond to attribute values for the node "`f`". In this case, an update
 or replace message specifying a path of `/a/f[k1=10][k2=20]` setting the value
-of `k1` to 100 MUST be considered erroneous, and a status code of
-`InvalidArgument (3)` specified.
+of `k1` to either the same value (10) or an alternate value (100) MUST be
```

Comment:
To support idempotent behavior, I believe setting a node which is also a list key to the same value it already has should be permitted.  Only changing the node's value must be an error.